packages/block-library/src/cover/block.json

```diff
@@ -53,6 +53,9 @@
 	"supports": {
 		"anchor": true,
 		"align": true,
+		"color": {
```

Comment:
The cover block does not use the supports mechanism for colors because it has some specificities (overlay) with this change the cover block has two color control UI's.

Reply:
Yep yep, not a real change. 👌